projects/systelab-components/src/lib/combobox/abstract-combobox.component.spec.ts

```diff
@@ -41,13 +41,13 @@ export class TestData {
 export class ComboboxTestComponent {
 	@ViewChild('combobox') public combobox: ModulabSelect;
 	public filter = false;
-	public valuesList: TestData[] = [new TestData('1', 'Description 1'), new TestData('2', 'Description 2')];
+	public valuesList: TestData[] = [new TestData(0, 'Description 0'),new TestData('1', 'Description 1'), new TestData('2', 'Description 2')];
```

Comment:
why 0 is a number? the rest of teh ids are characters with ' '

Reply:
The ID variable can be either a number or a string.
The number ID is the one that can raise with this error.